caiman/source_extraction/volpy/volparams.py

```diff
@@ -26,24 +26,25 @@ def __init__(self, fnames=None, fr=None, index=None, ROIs=None, weights=None, do
         }
 
         self.volspike = {
-            'doCrossVal': doCrossVal, # cross-validate to optimize regression regularization parameters
-            'doGlobalSubtract': doGlobalSubtract,
```

Comment:
It might be easier to read this dict if the keys are listed in order

Reply:
They are listed in order not alphabetically, but more follow the sequence of their appearance in the algorithm.